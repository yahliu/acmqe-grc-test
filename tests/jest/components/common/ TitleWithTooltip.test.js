/* Copyright (c) 2021 Red Hat, Inc. */
/* Copyright Contributors to the Open Cluster Management project */

'use strict'

import TitleWithTooltip from '../../../../src-web/components/common/TitleWithTooltip'

describe('TitleWithTooltip', () => {
  it('renders as expected for TitleWithTooltip', () => {
    const props = {
      className:'testingTitleWithTooltip',
      headingLevel:'h3',
      position:'top',
      title:'this is the title to test TitleWithTooltip',
      tooltip:'this is the tooltip to test TitleWithTooltip',
    }
    expect(TitleWithTooltip(props)).toMatchSnapshot()
  })
})
