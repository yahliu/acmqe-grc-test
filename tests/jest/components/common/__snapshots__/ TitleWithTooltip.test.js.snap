// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TitleWithTooltip renders as expected for TitleWithTooltip 1`] = `
<div
  className="testingTitleWithTooltip"
>
  <Title
    headingLevel="h3"
  >
    this is the title to test TitleWithTooltip
  </Title>
  <div
    className="GrayOutlinedQuestionCircleIcon"
  >
    <mockTooltip
      content={
        <div>
          this is the tooltip to test TitleWithTooltip
        </div>
      }
      position="top"
    >
      <OutlinedQuestionCircleIcon
        color="#6a6e73"
        noVerticalAlign={false}
        size="sm"
      />
    </mockTooltip>
  </div>
</div>
`;
